packages/material-ui/src/Rating/Rating.test.js

```diff
@@ -103,6 +103,17 @@ describe('<Rating />', () => {
     expect(checked.value).to.equal('2');
   });
 
+  it('should support highlight selected only', () => {
+    const value = 2;
+    const { container } = render(<Rating highlightSelectedOnly value={value} name="rating-test" />);
+    const checkedClass = container.querySelector(`label[for="rating-test-${value}"] > span`)
+      .classList[1];
+    const notCheckedClass = container.querySelector(`label[for="rating-test-${value - 1}"] > span`)
+      .classList[1];
```

Comment:
Please improve the expect API. We can't rely on the position of the class name in `classList`.